consensus/hotstuff/verification/combined_verifier_v2.go

```diff
@@ -118,9 +117,7 @@ func (c *CombinedVerifier) VerifyVote(signer *flow.Identity, sigData []byte, blo
 //   - model.ErrInvalidSignature if a signature is invalid
 //   - error if running into any unexpected exception (i.e. fatal error)
 func (c *CombinedVerifier) VerifyQC(signers flow.IdentityList, sigData []byte, block *model.Block) error {
-	if len(signers) == 0 {
-		return model.NewInsufficientSignaturesErrorf("empty list of signers")
-	}
+	msg := MakeVoteMessage(block.View, block.BlockID)
```

Comment:
curious why `msg` was moved to be constructed early and not right before the verification?

Reply:
🤷 don't know. You are right, having the construction of `msg` right before verification is most intuitive. done in https://github.com/onflow/flow-go/pull/3677/commits/be9459382d0a82a8fdcf9a04358ad133f325519d